Assets/Scripts/ObstacleRandomSprite.cs

```diff
@@ -0,0 +1,16 @@
+using System.Collections;
+using System.Collections.Generic;
```

Comment:
There are a few small changes you could make to clean up this script, but they're not a big deal:
- Remove unused using directives on lines 1 and 2.
- Remove comment above the `Start` method.
- Make the `Start` method `private`.
- Remove `gameObject.` from `gameObject.GetComponent`.